### PR TITLE
Adds prettier and open standup

### PR DESCRIPTION
**Pull request by orta**
Adds a note about how to run a standup.

### CODE REVIEW

File: meta/open_standup.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+### During standup
+
+* We start off with props, as a nod towards [People are Paramount][pplp]
+* Then we get an update from the CTO
+* Each team lead gives a summary of what their teams have been up to
+* We leave a spot for potential cross-team issues
+* We have a spot to talk about things people are proud of
+* We then either announce [the Lunch & Learn][ll], or try find one for the week
+
+During the standup, someone writes up these notes and passes them out on to slack after. Everyone leaves links to things
```
Comment (ashfurrow) [inline meta/open_standup.md]: Welllll, kind of. I usually listen for PRs or issues and find them afterwards myself, though sometimes people do leave them in the Slack. Worth mentioning?

**Review by ashfurrow — APPROVED** [review]

**Comment by dblock** [comment]
There needs to be a link to open-standup from README or somewhere else, too.

**Comment by orta** [comment]
Cool cool, addressed both feedbacks

File: LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2011-2016 Artsy Inc.
+Copyright (c) 2011-2017 Artsy Inc.
```
Comment (ashfurrow) [inline LICENSE]: 2018 now 😛

File: README.md [hunk 3]
```diff
@@ -46,6 +45,7 @@ A bunch of links for Engineers by Engineers.
 * [open engineering jobs](https://www.artsy.net/jobs#engineering)
 * [lunch and learn](meta/lunch_and_learn.md)
 * [happy hours](meta/happy_hour.md)
+* [open standup](meta/open_Standups.md)
```
Comment (ashfurrow) [inline README.md]: nitpick, but this is a capital `S`.

**Review by ashfurrow — APPROVED** [review]

**Pull request merged**